.github/workflows/sdk-breaking-change-labels.yaml

```
@@ -0,0 +1,55 @@
+name: SDK Breaking Change Labels (Preview)
+
+on:
+  check_run:
+    types: [completed]
+  workflow_dispatch:
+    inputs:
+      check_run_id:
+        description: 'Check run ID to process'
+        required: false
+        default: '38335408598'
+        type: string
+
```

```suggestion
permissions:
  contents: read
```

Start with these minimal permissions, and only add more if needed.